contracts/Package.sol

```diff
@@ -1,6 +1,9 @@
-pragma solidity 0.5.0;
+pragma solidity 0.5.7;
 
-contract Package {
+import "openzeppelin-solidity/contracts/ownership/Secondary.sol";
+
+
+contract Package is Secondary {
```

Comment:
Why do you use Secondary here? I read both Ownable and Secondary contracts and they are mostly the same.

Reply:
The only difference is that an owner can renounce ownership and a secondary [contract] can't. Neither fits perfectly but I've used the simpler one because the owner of a Package contract is another contract and not a person.